src/Wallabag/CoreBundle/Helper/EntriesExport.php

```diff
@@ -358,6 +360,25 @@ private function produceXML()
         )->send();
     }
 
+    private function produceTXT()
+    {
+        $content = '';
+        foreach ($this->entries as $entry) {
+            $content .= "\n\n" . str_repeat("=",100) . "\n\n" . $entry->getTitle() . "\n\n" . str_repeat("=",100) . "\n\n";
```

Comment:
You should put one `str_repeat("=", 100)` outside the `foreach`, store it in a variable and then re-use in the `foreach`. It'll be more efficient since you'll perform only one and only one `str_repeat`.